### ISSUE TITLE
각 학년 학기 노드의 위치 정렬

### ISSUE THREAD

**Issue by HeoSeungBeen** [oss2024hnu/coursegraph-py#482]
<img width="769" alt="스크린샷 2024-06-16 오전 12 34 56" src="https://github.com/oss2024hnu/coursegraph-py/assets/142968253/fd031f54-8ca4-4e20-bfc1-3e502c326335">

이런 형태의 그래프보다 우리가 일반적으로 보던 것 처럼
<img width="769" alt="스크린샷 2024-06-16 오전 12 34 56" src="https://github.com/oss2024hnu/coursegraph-py/assets/142968253/ff657c79-390b-4023-b094-5a796c28eea4">

이런 형태로 정렬 된 형태가 가독성 확보에 유리하다는 생각이듭니다

**Comment by kyagrd** [comment]
네 저도 그게 좋다고 생각하는데 그개 생각보다 어렵습니다. 이걸 절대 위치를 맘대로 정해주는 스타일의 엔진이 아니라서.
강제로 억지로 해봤는데 그러면 화살표가 점점 기괴한 모양 이 되더라고요.
이슈 자체는 타당하다고 저도 생각합니다. 기한이 얼마 남지 않아 이번 학기에 해결은 어렵겠지만요.
